src/Checker/OrderRefundingAvailabilityChecker.php

```diff
@@ -25,6 +25,9 @@ public function __invoke(string $orderNumber): bool
         $order = $this->orderRepository->findOneByNumber($orderNumber);
         Assert::notNull($order);
 
-        return $order->getPaymentState() === OrderPaymentStates::STATE_PAID;
+        return in_array(
+            $order->getPaymentState(),
+            [OrderPaymentStates::STATE_PAID, OrderPaymentStates::STATE_REFUNDED, OrderPaymentStates::STATE_PARTIALLY_REFUNDED]
```

Comment:
Does it mean that refunding is still available if the order payment is in state `Refunded`?

Reply:
That's because I wanted to make it possible to access the refund page when the order is fully refunded... but it won't be possible to refund anything because order's refunded total would be the same as order's total 😄 Probably the best choice would be to use a different service in a detection of possibility to go to the refund page and different for checking a refund action... On it.